content/python/concepts/regex/terms/metacharacters/metacharacters.md

```diff
@@ -0,0 +1,98 @@
+|      `.`      | Matches any character.                                           | `r'.'` matches "**code_ninja@codecademy.com**".                                                   |
+|     `[]`      | Matches character class inside the brackets not excluded by `^`. | `r'Char[mander\|meleon\|izard]'` matches "Char**mander**", "Char**meleon**", and "Char**izard**." |
+|      `^`      | Matches characters at the beginning of a string.                 | `r'^C'` matches "**C**odecademy."                                                                 |
+|      `$`      | Matches characters at the end of a string.                       | `r'y$'` matches "Codecadem**y**."                                                                 |
+
+## Quantifiers
+
+Some metacharacters are concerned with the frequency of certain character patterns as shown in the table below:
+
+| Metacharacter | Description                                                     | Example                                                                 |
+| :-----------: | --------------------------------------------------------------- | ----------------------------------------------------------------------- |
+|      `?`      | Matches zero or one of the preceding character.                 | `r'neighbo?ur'` matches "neighbor" and "neighbo**u**r."                 |
+|      `*`      | Matches zero or more of the preceding character.                | `r're*d'` matches "r**e**d" and "r**ee**d."                             |
+|      `+`      | Matches one of more of the preceding character.                 | `r'tw+o'` matches "t**w**o" but not "to."                               |
+|     `\|`      | Matches either the pattern before or after the `\|`.            | `r'true\|false'` matches "**true**" or "**false**."                     |
+|     `{x}`     | Matches if the preceding character occurred `x` times in a row. | `r're{2}d'` matches "r**ee**d" (2 "e"s) but not "r**e**d" (only 1 "e"). |
+
+## Capture Gropus
+
+Capture groups can be used to check and quantify different patterns in the string. This can be done with parentheses `(...)`, as shown in the example below:
+
+```py
+import re
+
+pattern_one = 'red'
+pattern_two = 'rad'
+pattern_three = 'rid'
+
+capture_group = r'r(e|a)d'
+
+print(re.match(capture_group, pattern_one))
+print(re.match(capture_group, pattern_two))
+print(re.match(capture_group, pattern_three))
+```
+
+The capture group uses the pipe `|` quantifer to match two out of the three patterns since they both have "e" or an "a". The following output will be printed:
+
+```shell
+<_sre.SRE_Match object; span=(0, 3), match='red'>
+<_sre.SRE_Match object; span=(0, 3), match='rad'>
+None
+```
+
+## Special Sequences
+
+The backslash `\` metacharacters has two primary uses:
+
+- Matches most characters next to it literally.
+- Search for a match within a sequence of particular metacharacters sucha as digits.
```

Comment:
```suggestion
- Search for a match within a sequence of particular metacharacters such as digits.
```

Reply:
I find these two points a bit confusing. Feels like the role of the backslash is being conflated with the defined meta characters which use a backslash. I think of the backslash as an independent operator that allows us to 'escape' or match literals that otherwise have another meaning such as parentheses, quotes or a dollar sign. The metacharacters are unique operators that aid in targeting specific character classes and act as shortcuts: so that we're not always specifying custom classes through enclosed brackets. That's my interpretation, happy to hear an alternative if I'm misreading your content or the Python regex documentation.